content/_includes/site/hire-us.njk

```diff
@@ -0,0 +1,27 @@
+{% import 'embed.macros.njk' as embed %}
+
+{%- set title = '<span class="title-action">Hire Us to Build</span> Custom Web Apps <span class="title-description">That stand the test of time</span>' -%}
```

Comment:
Should we keep Title Case throughout this whole line? I see the first  two lines are Title Case but the last line is sentence case. Is that intentional for emphasis?

Reply:
Yeah, I noticed that change on the second draft of the mockup @SondraE . Was that intentional?

Reply:
@stacyk @dvdherron That was intentional. The "Hire Us to Build" matches what we have at the top of the homepage and the sentence case of the last line matches what we have on blog posts. We can consider something different if it doesn't work here.